src/backend/base/langflow/api/v1/chat.py

```diff
@@ -197,12 +200,14 @@ async def build_flow(
         queue_service=queue_service,
         flow_name=flow_name,
     )
-    if settings_service.settings.event_delivery != "direct":
+
+    # This is required to support FE tests - we need to be able to set the event delivery to direct
+    if EventDeliveryType.DIRECT not in (settings_service.settings.event_delivery, event_delivery):
```

Comment:
No need to check in settings because the UI gets the value from the backend. What we can do is raise an error if `settings_service.settings.event_delivery != event_delivery`.
```suggestion

    # This is required to support FE tests - we need to be able to set the event delivery to direct
    if event_delivery == EventDeliveryType.DIRECT:
```